### ISSUE TITLE
App crashes on startup Oculus Quest

### ISSUE THREAD

**Issue by RaoulRb** [Funnyguy77/PocketSphinxUnityDemo#15]
Hi,

Thanks for getting pocketsphinx running in the first place!
Everything works great on pc both editor and builds. I am currently trying to get it to work on an oculus quest, as you mentioned this should work.
However on startup the app crashes. This seems to be the culprit:
 `Error AndroidRuntime:       #00 (wrapper managed-to-native) Pocketsphinx.pocketsphinxPINVOKE:Decoder_StartUtt (System.Runtime.InteropServices.HandleRef)`

However earlier in the crash log i find this log: `Error Unity: Unable to find libpocketsphinxwrap .`
Which seems like a problem.
I'm using unity version 2019.3.0f6.
I have attached the full crash log as well.
[PocketSphinxCrashQuest.txt](https://github.com/Funnyguy77/PocketSphinxUnityDemo/files/4623321/PocketSphinxCrashQuest.txt)

Thanks in advance for any help!
Kind regards

**Comment by RaoulRb** [comment]
Hey, So through further testing i found out that the problem was due to reading the text file with the keyphrases. 

This is a known issue on Oculus quest!
https://stackoverflow.com/questions/58063459/unity-streaming-assets-content-not-being-read-in-oculus-quest-go
https://forum.unity.com/threads/reading-streamingassets-data-fromapk-on-oculusquest-go.749858/

Because of this i was looking for a new way to set multiple keyphrases. But I am not sure how the text file is read inside pocketsphinx as  everything file related happens in the function below.
  //string keyphrasePath = Path.Combine(Application.streamingAssetsPath, "keywords.txt");
  //c.SetString("-kws", keyphrasePath);

Any workaround for this?

Kind regards

**Comment by Funnyguy77** [comment]
Hi there! 

Yes, this is a known issue for any file you attempt to load files on the Oculus Quest. I personally, haven't tried using multiple keyphrase spotting on the Quest. Usually I use language model spotting. 

Have you tried loading the text file as the StackOverflow thread you linked mentioned? 

Something like this could work: (Untested)

```
WWW reader = new WWW (keyphrasePath);
while (!reader.isDone) 
{
    yield return null;
}
keyphrases = reader.text;

c.SetString("-kws", keyphrases);
```

**Comment by geezlouisee** [comment]
Hi all, the sample scene worked great for me in the Unity Editor but now I am unable to deploy it successfully for Oculus Quest :( Similar to @RaoulRb , I also encountered a similar error `Unable to find libpocketsphinxwrap`.

The project is unable to open in the Oculus Quest after i loaded the apk in, but while i was running `adb.exe logcat`, some of the debug messages appear:
![image (1)](https://user-images.githubusercontent.com/39251616/85518859-4aa2ec00-b633-11ea-9691-8803dcbe4a66.png)

I've tried loading the text file mentioned in the StackOverflow thread as well as also tried downloading the "Better Streaming Assets" but none of those ways seem to solve the problem. 

Greatly appreciate it if anyone knows how they managed to work around this problem.

Thanks in advance!

PS. I am using Unity 2019.3.15f1.
